com.unity.formats.fbx/Tests/FbxApiTests/SampleCodeTest.cs

```diff
@@ -11,35 +11,18 @@ namespace FbxExporter.UnitTests
     /// <summary>
     /// Unit tests for sample code included in documentation.
     /// </summary>
-    public class SampleCodeTest
+    public class SampleCodeTest : ExporterTestBaseAPI
     {
-        private const string k_tempFolderName = "_safe_to_delete";
-        private string m_tempFolder = Path.Combine(Application.dataPath, k_tempFolderName);
-
-        [SetUp]
-        public void Init()
-        {
-            Assert.That(m_tempFolder, Does.Not.Exist);
-            Directory.CreateDirectory(m_tempFolder);
-        }
-
-        [TearDown]
-        public void Term()
-        {
-            if (Directory.Exists(m_tempFolder))
-            {
-                Directory.Delete(m_tempFolder, recursive: true);
```

Comment:
this was not deleting the directory properly, and so if you run the test again it would fail on init as the directory already exists. Reusing the code from the test base for creating/deleting directories